src/redux/constants/ActionTypes.js

```diff
@@ -5,3 +5,4 @@ export const SET_VIEWPORT = 'VIEWPORT::SET'
 export const SET_VIEWPORT_ACTIVE = 'VIEWPORT::SET_ACTIVE'
 export const SET_VIEWPORT_LAYOUT = 'VIEWPORT::SET_LAYOUT'
 export const CLEAR_VIEWPORT = 'VIEWPORT::CLEAR'
+export const SET_VIEWPORT_SPECIFIC_DATA = 'VIEWPORT::SET_VIEWPORT_SPECIFIC_DATA'
```

Comment:
To match the others, can we make the string value: `VIEWPORT::SET_SPECIFIC_DATA`

Where `VIEWPORT` is the scope/module, and what comes after the `::` is the action

Reply:
done